fapi-client/src/main/scala/com/gu/facia/api/utils/FaciaContentUtils.scala

```diff
@@ -136,7 +136,7 @@ object FaciaContentUtils {
   )
   def maybeShortUrl(fc: FaciaContent) = fieldsGet(fc)(_.flatMap(_.shortUrl))
   def shortUrl(fc: FaciaContent): String = maybeShortUrl(fc).getOrElse("")
-  def shortUrlPath(fc: FaciaContent) = maybeShortUrl(fc).map(_.replace("http://gu.com", ""))
+  def shortUrlPath(fc: FaciaContent) = maybeShortUrl(fc).map(_.replaceFirst("^[a-zA-Z]+://gu.com", ""))
```

Comment:
Would it be more robust to use a URL parser on this? You're after the "path". That way it'll work whatever changes around it (maybe at some point they'll start adding querystrings here, for example).

```scala
import java.net.URL

def shortUrlPath(fc: FaciaContent) = maybeShortUrl(fc).map(new URL(_).getPath)
```